sdk/src/index.ts

```diff
@@ -123,7 +123,7 @@ export interface MixinDeviceOptions<T> {
   get console() {
     if (!this._console) {
       if (deviceManager.getMixinConsole)
-        this._console = deviceManager.getMixinConsole(this.id, this.mixinProviderNativeId);
+        this._console = deviceManager.getMixinConsole(this.id!, this.mixinProviderNativeId);
```

Comment:
I just updated the sdk to make this type required, so can pull and remove this non-null assert.